jobs/scheduler/spec

```diff
@@ -57,6 +60,14 @@ properties:
     default: 3
     description: "Maximum number of notification sent to scaling engine for job re-schedule"
 
+  autoscaler.cf_server.xfcc.valid_org_guid:
+    description: approve org guid for xfcc endpoint
+    default: ''
+
+  autoscaler.cf_server.xfcc.valid_space_guid:
```

Comment:
I would following the convention already:
e.g
```suggestion
  autoscaler.scheduler.cf_server.xfcc.valid_space_guid:
```

Reply:
fixed 949502985